frontend/src/routes/root.route.tsx

```diff
@@ -8,7 +8,7 @@ export const router = createBrowserRouter([
     element: <NameEntry />,
   },
   {
-    path: ROUTES.QUESTIOANDANSWER,
+    path: `${ROUTES.QUESTIOANDANSWER}/:id`,
```

Comment:
**question (non-blocking):** can we add `/:id` to the ROUTES object `QUESTIOANDANSWER` value?

Reply:
The changes you are asking breaking the functionality.so, we will update the code in future if possible.